### PR TITLE
Fix minimumAge rule

### PR DESCRIPTION
**Pull request by KennedyTedesco**
This is related to #593

### CODE REVIEW

File: library/Rules/MinimumAge.php [hunk 1]
```diff
@@ -20,16 +21,16 @@ class MinimumAge extends AbstractRule
 
     public function __construct($age, $format = null)
     {
+        if (!filter_var($age, FILTER_VALIDATE_INT)) {
+            throw new ComponentException('The age must be a numeric value.');
```
Comment (henriquemoody) [inline library/Rules/MinimumAge.php]: This is great for version 1.1, but it will not me released on versions 0.8>.

File: library/Rules/MinimumAge.php [hunk 2]
```diff
@@ -20,16 +21,16 @@ class MinimumAge extends AbstractRule
...
+        }
+
         $this->age = $age;
         $this->format = $format;
     }
 
     public function validate($input)
     {
-        if (!is_int($this->age)) {
```
Comment (henriquemoody) [inline library/Rules/MinimumAge.php]: If you just change this line to `filter_var($age, FILTER_VALIDATE_INT)`, and send a pull request agains 0.8 branch, then we can release a path version with this fix, isn't that what you want?

Comment (KennedyTedesco) [inline library/Rules/MinimumAge.php]: Yes, this is exactly what I want. But it will also go to 1.0, right?

Comment (henriquemoody) [inline library/Rules/MinimumAge.php]: Yes, according to our [Release Cycle and Support](https://github.com/Respect/Validation/wiki/Release-Cycle-and-Support) we're still supporting 0.8, 0.9 and 1.0 as  well, of course.

Comment (henriquemoody) [inline library/Rules/MinimumAge.php]: But, you have to send another pull request against 0.8 branch to fix that on those versions.

Comment (KennedyTedesco) [inline library/Rules/MinimumAge.php]: Got it. I'll do it soon. :smile:

**Comment by henriquemoody** [comment]
After b98595a5c093bfebd8686646fffb6b441fce6a08 these changes leads with code duplication, can you rebase and change this code to fit the new codebase?

**Comment by KennedyTedesco** [comment]
Sure! I'll do it asap.

**Comment by henriquemoody** [comment]
Merged manually by cca733919c9f272ffe162b75fcc0043f5b93ea1a.

Thanks for contributing! 🐼
